RELEASING.md

```diff
@@ -342,33 +342,17 @@ Be sure to use "Rebase and Merge" when merging the PR to `main` to preserve the
 - [ ] Mention in the release thread on Discord that Dagger Cloud can be updated
       to the just-released version. cc @marcosnils @matipan @sipsma
 
-## 🐙 dagger-for-github ⏱ `10mins`
+## 🐙 dagger-for-github ⏱ `2mins`
```

Comment:
I wonder if we could just entirely remove this section, and move it to a `RELEASING.md` in `dagger-for-github` itself? Since it's entirely detached from this process now.

Reply:
Up to you! This currently LGTM as well :tada: